scripts/linters/pylint_extensions.py

```diff
@@ -1893,6 +1893,81 @@ def visit_compare(self, node):
                 self.add_message('inequality-with-none', node=node)
 
 
+class DisallowedFunctionsChecker(checkers.BaseChecker):
+    """Custom pylint checker for language specific general purpose
+    regex checks of functions calls to be removed or replaced.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+    name = 'disallowed-function-calls'
+    priority = -1
+    msgs = {
+        'C0031': (
+            'Please do not use %s.',
+            'remove-disallowed-function-calls',
+            'Please remove disallowed function calls.',
```

Comment:
The 1st param should be a user-facing message for the developer so it should be a simple action for them to do (maybe `Please remove %s function call.` [or something around that])

The 3rd param should be a user-facing message for the developer trying to check what this check class does [using pylint --help] (maybe `Disallows usage of black-listed functions.` [or something around that]) 

Ditto for other messages below*

Reply:
Done!